Resources/Prototypes/Entities/Structures/Wallmounts/timer.yml

```diff
@@ -0,0 +1,64 @@
+- type: entity
+  id: SignalTimer
+  name: signal timer
+  description: It's a timer for sending timed signals to things.
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Wallmount
+  components:
+  - type: Transform
+    anchored: true
+  - type: WallMount
+    arc: 360
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Sprite
+    sprite: Structures/Wallmounts/switch.rsi
+    netsync: false
+    state: on
+  - type: Appearance
+  - type: Rotatable
+  - type: Fixtures
+  - type: SignalTimer
+    canEditLabel: false
+  - type: SignalTransmitter
+    outputs:
+      Start: []
+      Timer: []
+  - type: ActivatableUI
+    key: enum.SignalTimerUiKey.Key
+  - type: UserInterface
+    interfaces:
+    - key: enum.SignalTimerUiKey.Key
+      type: SignalTimerBoundUserInterface
+  - type: ApcPowerReceiver
+    powerLoad: 100
+  - type: Electrified
+    enabled: false
+    usesApcPower: true
+  - type: ExtensionCableReceiver
+  - type: ActivatableUIRequiresPower
+      
+- type: entity
+  id: ScreenTimer
+  parent: SignalTimer
+  name: screen timer
+  description: It's a timer for sending timed signals to things, with a built-in screen.
+  components:
+  - type: SignalTimer
+    canEditLabel: true
+  - type: TextScreenVisuals
+  - type: Sprite
+    sprite: Structures/Wallmounts/textscreen.rsi
+    state: textscreen
+    noRot: true
+
+- type: entity
+  id: BrigTimer
+  parent: ScreenTimer
+  name: brig timer
+  description: It's a timer for brig cells.
+  components:
+  - type: AccessReader
+    access: [["Brig"]]
```

Comment:
this should be sec access so lawyers cant just free their buddies using the timer